tslearn/barycenters/dba.py

```diff
@@ -304,13 +317,15 @@ def _mm_valence_warping(list_p_k, barycenter_size, weights):
         list_p_k=list_p_k,
         barycenter_size=barycenter_size,
         weights=weights)
-    diag_sum_v_k = numpy.zeros(list_v_k[0].shape)
-    for v_k in list_v_k:
-        diag_sum_v_k += v_k
+    #diag_sum_v_k = numpy.zeros(list_v_k[0].shape)
+    #for v_k in list_v_k:
+    #    diag_sum_v_k += v_k
+    diag_sum_v_k = Parallel(backend='threading',prefer='threads',require='sharedmem', n_jobs=n_treads, verbose = True)(delayed(numpy.sum)(x) for x in zip(*numpy.array(list_v_k)))
+    diag_sum_v_k = numpy.array(diag_sum_v_k)
     return diag_sum_v_k, list_w_k
 
 
-def _mm_update_barycenter(X, diag_sum_v_k, list_w_k):
+def _mm_update_barycenter(X, diag_sum_v_k, list_w_k, n_treads):
```

Comment:
We use `n_jobs` everywhere as a parameter name, and `None` as its default, we should stick to the same convention here, I think.